app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError,
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("All friends should be vaccinated")
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("Outdata vaccine")
+        if (
+            "wearing_a_mask" not in visitor
+            or visitor["wearing_a_mask"] is False
+        ):
+            raise NotWearingMaskError("All friends must be wear masks")

```

Comment:
The error message in the `NotWearingMaskError` exception is not very clear. It would be better to include more details about the error. For example, 'The visitor is not wearing a mask. All visitors should wear a mask.'